src/makers/example/hotspots.ts

```diff
@@ -2,12 +2,14 @@ import HotspotIcon from './icon.svg'
 import { MakerHotspot } from '../hotspotMakerTypes'
 import ANTENNAS from './antennas'
 
+// TODO: variants
```

Comment:
@vpetersson @shawaj we must define all the miner types we support.

- Indoor (Raspberry Pi)
- Outdoor (Raspberry Pi)
- Indoor (ROCK Pi)

What other variants do we want to include in the app? What are the public facing names for these variants? The labels currently say something like "Nebra Helium Hotspot: NEBHNT-HHRK4-915", none of which makes sense to use in this context.